docs/_partners/integration-quality.md

```diff
@@ -0,0 +1,126 @@
+* Can’t keep up with maintenance? Hire a Zapier expert to help you fix on-off bugs or maintain your integration.
+* Make sure the right people on your team are getting issue notifications. You can add emails to receive these under the [“Notification Settings”](https://cdn.zappy.app/1b35f51a79c3caead06fedeb1fe0cffb.png) of the Bugs & Feature Requests tab in the integration’s developer platform.
+
+
+## Active Users Retention
+
+At Zapier, churn means a user used your integration in their Zaps 29 - 56 days ago, but hasn’t run a successful task in one of those Zaps in the past 28 days. This user is considered to have churned from the integration. From the opposite perspective, we can look at active users retention, or the percentage of users who haven’t churned.
+
+### Did they leave Zapier altogether?
+
+Not necessarily. Maybe they switched to using a competing integration or their workflow had a more periodic or seasonal cadence.
+
+But, it could also mean they got so frustrated with the experience of trying to get their Zap working and _keep_ it working successfully, they turned it off, deleted it, and walked away.
+
+### What we have learned
+
+Lowered active user retention rates don’t necessarily mean poor integration health. Some apps lend themselves to use-cases with shorter life-spans than others. That said, spikes in churn rate not related to seasonal variations in usage could be indicators of a problem with something not functioning as expected.
+
+Active user retention is not a leading indicator. In fact, it’s quite a lagging one that may only start indicating a problem up to 28 days after it has been an issue. That doesn’t deem it useless, we just have to know how to make full use of it
+
+### Best Practices
+
+Approaching active user retention with a long-term strategy can help maintain a consistently high level of retention:
+ 
+* [Embed](https://platform.zapier.com/embed/overview) the Zapier experience with copy-and-paste and customizable code within your platform to provide automation value directly to users. Embeds have proven to [reduce churn](https://platform.zapier.com/partner_success_stories/all) on Partners’ platforms. Find live embed examples under the ‘Embed’ tab of the integration’s developer platform.
+* [Share use cases](https://platform.zapier.com/partners/partner-faq#tip-3-share-zapier-use-cases-in-your-onboarding) widely during your platform’s onboarding process. Having multiple Zaps using your integration increases stickiness of users not only to the Zapier integration, but also to your platform.
+* Update the integration regularly with features as your platform evolves. Invite stakeholders as [Collaborators](https://platform.zapier.com/partners/faq#add-collaborators-to-your-team) to your integration to give them read-only access to insights, metrics, and feedback to prioritize and align improvements. 
```

Comment:
Should we link this to the new [`Invite Team Member`](https://platform.zapier.com/quickstart/invite-team-member) page instead, since it has the instructions on how to invite?